### PR TITLE
More refs for Mars Climate Orbiter

### PR DESCRIPTION
**Pull request by BigEd**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -149,9 +149,9 @@ Sun/Oracle. Sun famously didn't include ECC in a couple generations of server pa
 
 [NASA](http://www.doneyles.com/LM/Tales.html). A design flaw in the Apollo 11 rendezvous radar produced excess CPU load, causing the spacecraft computer to restart during lunar landing.
 
-[NASA](https://en.wikipedia.org/wiki/Mars_Climate_Orbiter). Use of different units of measurement (metric vs. English) caused Mars Climate Orbiter to fail. This is basically the same issue Sweden ran into in 1628 with its ship.
+[NASA](https://en.wikipedia.org/wiki/Mars_Climate_Orbiter). Use of different units of measurement (metric vs. English) caused Mars Climate Orbiter to fail. (This is basically the same issue Sweden ran into in 1628 with its ship.) There were also organisational and procedural failures[[ref](http://space.stackexchange.com/a/20241)] and defects in the navigation software[[ref](http://spectrum.ieee.org/aerospace/robotic-exploration/why-the-mars-probe-went-off-course)].
```
Comment (kerrizor) [inline README.md]: Might I suggest dropping the parenthetical reference to the sinking of the _Vasa_? It is listed here, but much further down the file, and without anchor linking it feels like a distraction.

Comment (BigEd) [inline README.md]: That would work for me. I wonder if @danluu is fond of the cross-reference. Dan?

Comment (danluu) [inline README.md]: Sorry for the slow response, but dropping that SGTM. I believe that's a relic from when this document was much smaller.

**Comment by BigEd** [comment]
OK, removed that parenthetical remark.
Note that my change also includes a typo fix: pathfinder/pathfilder - probably I should have resisted the temptation to fix that in the same change.

**Comment by danluu** [comment]
Thanks, as always :-)

**Pull request merged**